proposals/physical-cameras/README.md

```diff
@@ -314,4 +314,94 @@ class PhysicalCameraLensModelPinhole "PhysicalCameraLensModelPinhole" (
         doc = '''squeeze'''
     )
 }
+
+class PhysicalCameraExposure "PhysicalCameraExposure" (
+    doc = """
+    Describes the exposure settings common to still and cinema cameras.
+
+    These settings are used to scale the brightness of the illumination incident
+    on the camera system. The scaling factor, S, is computed as:
+       S = sensitivity_factor * iso * 2^exposure_compensation / (100 * shutterspeed * fnumber^2)
+    """
+    inherits = </APISchemaBase>
+    customData = {
+        token apiSchemaType = "singleApply"
+    }
+)
+{
+    float physcam:exposure:fnumber = 1.0 (
+        doc = """The lens aperture f-number"""
+    )
+
+    float physcam:exposure:iso = 100 (
+        doc = """The speed rating of the sensor or film"""
+    )
+
+    float physcam:exposure:shutterspeed = 1.0 (
+        doc = """The exposure time as fractions of a second, e.g. a value of 
```

Comment:
i.e. should be deleted from the body schema section.

Reply:
I go with shutter speed really since that's what I've always used and that's how it's typically expressed when talking about exposure from what I've seen, particularly since that's how it's normally displayed on a stills camera.

As for the duplication of parameters in different places for exposure and lens/body, yeah in some senses it's ugly, but in practical terms it's incredibly useful. IME you often want to express the brightness of an image separately from the geometry of its construction. As in, the director says "make the background blurrier" or "less motion blur" and you want to be able to control that without then compensating the brightness of your image.

Reply:
That's a great insight. I wonder if that note can be recorded into this document for future reference as to why it appears in two places? Also, I think that implies  we should leave the `duration` alone on the body, since that one would be the geometrical aspect of it, and physics etc, are going to be using seconds?

Reply:
Yep I think we leave all the body parameters as-is. In my mind, this is a schema you'd apply at scene-authoring time in order to provide and communicate creative choices, while the values in the body and lens model would be used for transmitting data from set. Your pipeline may automatically add this schema when constructing a shot, using the values from the shoot, or it may rely purely on the shoot values and never use this schema (if you want to be _really_ locked down), or you may not use the rest of PhysicalCamera at all and just use this schema to provide real-world exposure controls to a regular camera (this is the route we'll use most of the time in Omniverse).

Reply:
That makes sense!

Reply:
> As for the duplication of parameters in different places for exposure and lens/body, yeah in some senses it's ugly, but in practical terms it's incredibly useful. IME you often want to express the brightness of an image separately from the geometry of its construction. As in, the director says "make the background blurrier" or "less motion blur" and you want to be able to control that without then compensating the brightness of your image.

I think in V-Ray the "automatic exposure" just shifts the ISO gain without changing the lens aperture F-Stop and Shutter speed which technically is AutoISO then, not AE.  That is adjusting brightness without changing motion blur and depth of field. There is AE (Automatic Exposure) in UE now but I am not clear what it controls... 

 ISO (gain in linear space) and Shutter Speed are part of the camera body and Lens Aperture of the lens. So exposure (exposure compensation variables) is sort of a function of lens-camera body pairing, in that way, a bit like FOV is a function of pairing a sensor capture area and a lens focal length.  I say that just to make clear distinction between matching a real camera and color or tone correction so it looks right to you ("creative"). Which I think you also say. Except I still don't understand why you need shutter speed distinct in some exposure controls than the base shutter opening. You want same net exposure with motion blur or off? 
 
 Also for our purpose what is the difference between iris and aperture, we just need the aperture f/number no? Isn't the same value for us?
 
 Is this exposure schema really just highlighting need for a camera system level shader model, things not addressed by material shaders. One could make another schema arriving to a similar result using black level/offset, lift, and gain (in linear space)... no?  Then another one will say what about the two compensation variables for white balance...